data/brands/amenity/fast_food.json

```
@@ -3208,6 +3210,35 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "KFC",
```

```suggestion
      "displayName": "KFC (India & Pakistan)",
```
different displayName may cause conflict, so this should be unique